packages/api/package.json

```
@@ -11,15 +11,20 @@
     "publish": "wrangler publish --env $(whoami)",
     "build": "webpack",
     "format": "prettier --write '**/*.{js,css,json,md}'",
-    "test": "playwright-test \"test/**/*.spec.js\" --sw src/index.js"
+    "test": "npm run test:e2e",
+    "test:e2e": "npm-run-all -p -r test:e2e:**",
+    "test:e2e:mock:db": "smoke -p 9086 test/mocks/db",
+    "test:e2e:playwright": "playwright-test \"test/**/*.spec.js\" --sw src/index.js"
```

minor: if it so pleases you, I think the targets in the client are a little more pleasing. Not much but it would be good to have a similar pattern in both, either way.
```suggestion
    "test": "npm-run-all -p -r mock:api test:e2e",
    "test:e2e": "playwright-test \"test/**/*.spec.js\" --sw src/index.js"
    "mock:db": "smoke -p 9086 test/mocks/db",
```

---

Yes lets do it that way.